.spyproject/codestyle.ini

```diff
@@ -0,0 +1,6 @@
+[codestyle]
```

Comment:
Should remove these .spyproject files from this PR. I saw your other PR put them into the .gitignore which is good. /

I'll add a comment to this pull request describing how to revert the unwanted file changes.